proparse/src/main/java/org/prorefactor/proparse/antlr4/ProgressLexer.java

```diff
@@ -1002,6 +1002,20 @@ public int getCol() {
     public int getSourceNum() {
       return sourceNum;
     }
+
+    @Override
+    public int hashCode() {
+      return (13 * file) + (17 * line) + (31 * col) + (37 * sourceNum);
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+      if (obj instanceof FilePos) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Compare to "this.getClass()" instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS2162)